tests/wicked/startandstop_sut.pm

```diff
@@ -47,16 +47,32 @@ use mmapi;
 sub run {
     my ($self) = @_;
     my %results;
+    my $config;
+    my $dummy;
     my $iface = iface();
 
-    $self->before_scenario('Test 1', 'Bridge - ifreload');
-    my $config = '/etc/sysconfig/network/ifcfg-br0';
-    # Do actions here
-    # $results{1} = $self->get_test_result("ifcfg-br0", "");
-    $results{1} = 'PASSED';
+    $self->before_scenario('Test 1', 'Bridge - ifreload', $iface);
+    $config = '/etc/sysconfig/network/ifcfg-br0';
+    $dummy  = '/etc/sysconfig/network/ifcfg-dummy0';
+    $self->get_from_data('wicked/ifcfg/br0',    $config);
+    $self->get_from_data('wicked/ifcfg/dummy0', $dummy);
+    $self->setup_bridge($config, $dummy, 'ifreload');
+    $results{1} = $self->get_test_result("br0", "");
     mutex_create("test_1_ready");
-    #$self->cleanup($config, "ifcfg-br0");
+    $self->cleanup($config, "br0");
+    $self->cleanup($dummy,  "dummy0");
 
+    $self->before_scenario('Test 2', 'Bridge - ifup, ifreload', $iface);
+    $config = '/etc/sysconfig/network/ifcfg-br0';
+    $dummy  = '/etc/sysconfig/network/ifcfg-dummy0';
+    $self->get_from_data('wicked/ifcfg/br0',    $config);
```

Comment:
what's the point to do it second time ?

Reply:
cause the tests modify the content of br0 with sed command and we need a fresh br0 config file to be able to do the same thing again and again.
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/wickedbase.pm#L129